app/routes/circulars/circulars.server.ts

```diff
@@ -120,33 +176,10 @@ export async function search({
   } = await client.search({
     index: 'circulars',
     body: {
-      query: {
-        bool: {
-          must: query
-            ? {
-                multi_match: {
-                  query,
-                  fields: ['submitter', 'subject', 'body'],
-                },
-              }
-            : undefined,
-          filter: {
-            range: {
-              createdOn: {
-                gte: startTime,
-                lte: endTime,
-              },
-            },
-          },
-        },
-      },
+      query: esQuery,
       fields: ['subject'],
       _source: false,
-      sort: {
-        circularId: {
-          order: 'desc',
-        },
-      },
+      sort: { _score: 'desc' },
```

Comment:
If there is no query text, then the matches should be in descending chronological order.

If there is a query text, then it's OK to rank them by descending relevance.

Reply:
got this.